db/re/item_db_usable.yml

```diff
@@ -689,7 +688,7 @@ Body:
     Flags:
       BuyingStore: true
     Script: |
-      itemheal 200,0;
+      itemheal rand(175,234),0;
```

Comment:
It was fine according to aegis data and there is no value in kro description, please revert

Reply:
Thanks !